appveyor.yml

```diff
@@ -51,7 +51,26 @@ build_script:
   - 7z a grcov-win-%PLATFORM_TARGET%.tar.bz2 grcov-win-%PLATFORM_TARGET%.tar
 
 test_script:
+  - ps: echo $env:CHANNEL
+  - ps: |
+      If ($env:CHANNEL -eq "nightly") {
+         $env:CXX = "clang-cl.exe"
+         $env:CXXFLAGS = "--coverage"
+         $env:CARGO_INCREMENTAL = "0"
+         $env:RUSTFLAGS = "-Zprofile -Ccodegen-units=1 -Ctarget-feature=+crt-static"
+         cargo clean
```

Comment:
Instead of cleaning and rebuilding, we should set these env variables before the build